store/sqlstore/scheme_supplier.go

```diff
@@ -26,8 +26,10 @@ func initSqlSupplierSchemes(sqlStore SqlStore) {
 		table.ColMap("Scope").SetMaxSize(32)
 		table.ColMap("DefaultTeamAdminRole").SetMaxSize(64)
 		table.ColMap("DefaultTeamUserRole").SetMaxSize(64)
+		table.ColMap("DefaultTeamGuestRole").SetMaxSize(64)
```

Comment:
I'm adding this columns as part of the migration, but I guess is not a problem to have them duplicated here

Reply:
Yes, I think you are meant to put them here too.